src/pages/settings/Subscription/utils.ts

```diff
@@ -19,4 +19,11 @@ function getNewSubscriptionRenewalDate(): string {
     return format(startOfMonth(addMonths(new Date(), 12)), CONST.DATE.MONTH_DAY_YEAR_ABBR_FORMAT);
 }
 
-export {getNewSubscriptionRenewalDate, formatSubscriptionEndDate};
+function isCardExpired(expiryMonth: number, expiryYear: number): boolean {
```

Comment:
Do we need duplicate methods? I can see it's defined in DateUtils as well?

Reply:
Oh yeah, my bad, forgot to remove that. I was thinking about putting it in local utils but thought that there might be more places where this util can be used so I've moved it